modules/json/filterx-cache-json-file.c

```diff
@@ -209,36 +250,75 @@ _deep_freeze(FilterXFuntionCacheJsonFile *self, FilterXObject *object)
 
   object = filterx_ref_unwrap_ro(object);
   if (filterx_object_is_type(object, &FILTERX_TYPE_NAME(json_object)))
-    _deep_freeze_dict(self, object);
+    _deep_freeze_dict(self, object, root_object);
```

Comment:
This commit contains multiple changes:
- renaming types
- refactoring function signatures